Three/include/CGAL/Three/Scene_item.h

```diff
@@ -231,8 +231,21 @@ class SCENE_ITEM_EXPORT Scene_item : public QObject {
 
   //!Handles key press events.
   virtual bool keyPressEvent(QKeyEvent*){return false;}
+
+  //!Contains the header for the table in the statistics dialog
+  struct header_data{
+   //!Contains the name of the category of statistics and the number of lines it will contain
+   QList<std::pair<QString, int> > categories;
+   //!Contains the name of the lines of each category. Must be sorted from top to bottom.
+   QList<QString> titles;
+  };
+  //!Fills a header_data struct with the header information.
+  virtual void header(header_data&);
+  //!Returns a QString containing the requested value for the the table in the statistics dialog
+  virtual QString compute_stats(int i);
   //!Contrains the number of group and subgroups containing this item.
```

Comment:
I do not think that the documentation is sufficient. A reader of that documentation would not know what to write in a new class of scene item.